sw/otbn/crypto/p256.s

```diff
@@ -1031,7 +1016,7 @@ scalar_mult_int:
     jal       x1, proj_add
 
     /* probe if MSb of either one or both of the two
```

Comment:
nit: Can comments approximate to 80cols width? I think in some cases it is ok if not, if we want an equation to not spill into multiple lines for example.